Add more details to the lock machines page (#387)
statelyai/docs: docs/lock-machines.mdx

title: Lock machines
---

> You can lock a machine anytime to prevent accidental edits. Locked machines have a <Lock size={18} /> **Locked** status displayed in the editor’s top bar. They also have a lock icon in the project’s machine list in the left panel.

Reviewer: ```suggestion
You can lock a machine anytime to prevent accidental edits. Locked machines have a <Lock size={18} /> **Locked** status displayed in the editor’s top bar and in the project’s machine list in the left panel.
```

From the machine’s <Info size={18} /> **Details** panel, use the <Unlock size={18} /> Lock machine / <Lock size={18} /> Unlock machine icon button.

> If the machine is already locked, you can unlock it by clicking on the **Locked** button in the editor’s top bar and then confirm that you wish to unlock the machine.

Reviewer: ```suggestion
If the machine is already locked, you can unlock it by clicking on the **Locked** button in the editor’s top bar and then confirming that you wish to unlock the machine.
```
